RoboJackets/urc-software
# Docker Drone Installation Instructions

# Description
This PR does the following:
- Update docker install instructions 
- Fix  `Couldn't find directory for external packages...` and `Couldn't find .git folder in external package external/nanopb/.git ...` CMake errors

# Testing Steps
1. Follow docker installation instructions using manual build method (someone with a mac should try as well) 
3. `colcon build` urc drone and urc rover
4. check that gui works (try `gazebo` and `rviz2`)

Expectation: everything works

# Self Checklist
